src/crypto/tls/cfkem.go

```diff
@@ -58,6 +64,8 @@ func curveIdToCirclScheme(id CurveID) kem.Scheme {
 		return hybrid.Kyber768X25519()
 	case P256Kyber768Draft00:
 		return hybrid.P256Kyber768Draft00()
+	case DummyKex:
+		return hpke.KEM_X25519_HKDF_SHA256.Scheme()
```

Comment:
What is the intended semantics of this? Are we expected to negotiate it? Can we have a test in cfkem_test.go for this identifier?

Reply:
Added test. Semantics is as explained in the comment of the CurveID.

Reply:
Thanks, the test at least assures that we won't crash.